src/Symfony/Component/DependencyInjection/ParameterBag/ParameterBag.php

```diff
@@ -264,6 +219,9 @@ public function escapeValue($value)
         return $value;
     }
 
+    /**
+     * {@inheritdoc}
+     */
     public function unescapeValue($value)
```

Comment:
It seems that it was mostly introduced in https://github.com/symfony/symfony/pull/4707. 
We can still add a new argument to ``resolveValue()`` which would specify if the value must be unescaped (and change its default value in 4.0) and deprecate ``unescapeValue`` and ``escapeValue``.